LLMonFHIR/FHIR Display/MultipleResourcesChatView.swift

```diff
@@ -25,27 +24,30 @@ struct MultipleResourcesChatView: View {
     @AppStorage(StorageKeys.enableTextToSpeech) private var textToSpeech = StorageKeys.Defaults.enableTextToSpeech
     
     
-    private var disableInput: Binding<Bool> {
-        Binding(
-            get: {
-                multipleResourceInterpreter.viewState == .processing
-            },
-            set: { _ in }
-        )
-    }
-    
     var body: some View {
         @Bindable var multipleResourceInterpreter = multipleResourceInterpreter
         NavigationStack {
-            ChatView($multipleResourceInterpreter.chat, disableInput: disableInput)
+            Group {
+                if let llm = multipleResourceInterpreter.llm {
+                    let contextBinding = Binding { llm.context } set: { llm.context = $0 }
```

Comment:
Would it be possible to sue [`@Bindable`](https://developer.apple.com/documentation/swiftui/bindable) here to create the binding of the Observable type?

Reply:
Sadly not, I couldn't get that to work in my tries. The optionality of the `llm` is cruicial